docs(gen): tutorials requeriments update part ii (#2924)
scaleway/docs-content: tutorials/django-ubuntu-focal-fossa/index.mdx

>     The Django project you've created provides the structural basis for designing a more complete site. Check out the Django documentation for more information about how to build your applications and customize your site.

Reviewer: ```suggestion
    The Django project you have created provides the structural basis for designing a more complete website. Check out the Django documentation for more information about how to build your applications and customize your website.
```
